userguide/services.rst

```diff
@@ -224,41 +224,41 @@ service in :menuselection:`Services --> Dynamic DNS`.
-   |                       |                   | and enter the information in the *Custom Server* and *Custom Path* fields.                                      |
-   |                       |                   |                                                                                                                 |
+   |                       |                   | and enter the information in the :guilabel:`Custom Server` and :guilabel:`Custom Path` fields.                  |
    +-----------------------+-------------------+-----------------------------------------------------------------------------------------------------------------+
-   | CheckIP Server SSL    | string            | Set to use HTTPS for the connection to the *CheckIP Server*.                                                    |
-   |                       |                   |                                                                                                                 |
+   | CheckIP Server SSL    | checkbox          | Use HTTPS for the connection to the *CheckIP Server*.                                                           |
    +-----------------------+-------------------+-----------------------------------------------------------------------------------------------------------------+
-   | CheckIP Server        | string            | Enter the name and port of the server that reports the external IP address. Example: *server.name.org:port*.    |
-   |                       |                   |                                                                                                                 |
+   | CheckIP Server        | string            | Enter the name and port of the server that reports the external IP address. For example, entering               |
+   |                       |                   | :samp:`checkip.dyndns.org:80` uses `Dyn IP detection <https://help.dyn.com/remote-access-api/checkip-tool/>`__  |
+   |                       |                   | to discover the remote socket IP address.                                                                       |
    +-----------------------+-------------------+-----------------------------------------------------------------------------------------------------------------+
-   | CheckIP Path          | string            | Enter the path that is requested by the *CheckIP Server* to determine the user IP address.                      |
-   |                       |                   |                                                                                                                 |
+   | CheckIP Path          | string            | The path to the :guilabel:`CheckIP Server`. For example, the *no-ip.com* provider uses the                      |
+   |                       |                   | :samp:`/checkip.html` path for the :samp:`dynamic.zoneedit.com` :guilabel:`CheckIP Server`.                     |
```

Comment:
The multiple clauses in the example make it kind of hard to tell which value is entered where.  Does it mean that for no-ip.com, CheckIP Server is set to dynamic.zoneedit.com and CheckIP Path is set to /checkip.html?